setup/set_up_windows.md

```diff
@@ -4,168 +4,164 @@ This document provides the steps to install the software required to begin the e
 
 1.	Install Java JDK 1.8
 2.	Edit the JAVA_HOME variable and edit the system PATH
-3.	Install GIT
-4.	Install cURL
-5.	Install node.js
-6.	Install API Connect
+3.  Install Gradle
+4.	Install GIT
+5.	Install cURL
+6.	Install node.js
 7.	Install Docker
-8.	Install cloudfoundry CLI
-9.	Install the IBM Container plugin for cloudfoundry
+8.	Install IBM Cloud (BLuemix) CLI
+9.	Install the IBM Container Service plugins
+10. Install the Kubernetes kubectl CLI
+11. Install the MYSQL client
+12. Install Cygwin
+
 
 ## Part 1: Install Java JDK 1.8
 
 1.	In a browser, open https://www.oracle.com
```

Comment:
Any chance we can use the IBM JDK?  Not sure if Oracle's will cause a problem for students who install Eclipse and WAS for other classes.